File: experiments/020-PythonVsJava/cells/java-gradle_1-Empty_free/2019-2c-parcial2-masked/20260921-214358-38088/analysis/own-tests-project/src/main/java/surveycrawler/ClockwiseTurn.java
package surveycrawler;

public class ClockwiseTurn extends CrawlerTurn {

    // applying

    @Override
    public void applyTo(SurveyCrawler aCrawler) {
        aCrawler.turnFacingClockwise();
    }

    // undoing

    @Override
    public CrawlerAction opposite() {
        return CrawlerTurn.counterClockwise();
    }
}

File: experiments/020-PythonVsJava/cells/python-pytest_1-Empty_free/2019-2c-parcial2-masked/20260921-202806-59410/analysis/acceptance-project/src/surveycrawler/clockwise_turn.py
from surveycrawler.crawler_turn import CrawlerTurn


class ClockwiseTurn(CrawlerTurn):

    # turning

    def apply_to(self, a_crawler):
        a_crawler.facing().turn_clockwise(a_crawler)

    def inverted(self):
        from surveycrawler.counter_clockwise_turn import CounterClockwiseTurn

        return CounterClockwiseTurn()
